content/en/docs/setup/production-environment/tools/kops.md

```diff
@@ -213,23 +223,20 @@ for production clusters!
 
 ### Explore other add-ons
 
-See the [list of add-ons](/docs/concepts/cluster-administration/addons/) to explore other add-ons, including tools for logging, monitoring, network policy, visualization &amp; control of your Kubernetes cluster.
+See the [list of add-ons](/docs/concepts/cluster-administration/addons/) to explore other add-ons, including tools for logging, monitoring, network policy, visualization, and control of your Kubernetes cluster.
 
 ## Cleanup
 
 * To delete your cluster: `kops delete cluster useast1.dev.example.com --yes`
 
-## Feedback
-
-* Slack Channel: [#kops-users](https://kubernetes.slack.com/messages/kops-users/)
-* [GitHub Issues](https://github.com/kubernetes/kops/issues)
-
 {{% /capture %}}
 
 {{% capture whatsnext %}}
 
 * Learn more about Kubernetes [concepts](/docs/concepts/) and [`kubectl`](/docs/user-guide/kubectl-overview/).
 * Learn about `kops` [advanced usage](https://github.com/kubernetes/kops)
 * See the `kops` [docs](https://github.com/kubernetes/kops) section for tutorials, best practices and advanced configuration options.
+* Follow `kops` community discussions on the Slack Channel: [#kops-users](https://kubernetes.slack.com/messages/kops-users/)
```

Comment:
@kihahu , Some more suggestions 😃 
Consolidate lines 237 & 238. Both lines link to the project repo.
Line 239 could point to the project github page, https://github.com/kubernetes/kops#other-ways-to-communicate-with-the-contributors instead of the slack channel. The idea is to let the project manage the links/contents for the project.
Line 240 is OK, or you could link to: https://github.com/kubernetes/kops#github-issues.

Reply:
Done the updates as recommended, take a look.